### PR TITLE
Allow Git Blame overhead measurement in BlameReportExperiment

### PR DESCRIPTION
**Pull request by boehmseb**
- Adds possibility to adapt existing steps to work with ZippedExperimentSteps
- Inject blame measurement flag in BlameReportExperiment compile step

### CODE REVIEW

**Review by vulder — APPROVED** [review]

File: varats/varats/experiments/vara/blame_report_experiment.py [hunk 1]
```diff
@@ -99,18 +102,32 @@ def analyze(self) -> actions.StepResult:
...
+        return actions.StepResult.OK
+
+
 class BlameReportExperiment(VersionExperiment, shorthand="BRE"):
     """Generates a blame report of the project(s) specified in the call."""
 
     NAME = "GenerateBlameReport"
 
-    REPORT_SPEC = ReportSpecification(BR)
+    REPORT_SPEC = ReportSpecification(BR, WallTimeReportAggregate)
```
Comment (vulder) [inline varats/varats/experiments/vara/blame_report_experiment.py:124]: Do we want to keep this?

**Review by vulder — CHANGES_REQUESTED** [review]
Overall, everything looks nice and tidy.

The only thing we should discuss is: Should we keep the WTReport for BlameReports? As we don't plan on expanding/by default using the measurement infra on the vara side.

File: varats/varats/experiments/vara/blame_report_experiment.py (hunk 1 above)
Comment (boehmseb) [reply varats/varats/experiments/vara/blame_report_experiment.py:124]: I don't think we really need this. But we could maybe leave this branch lying around somewhere,

Comment (vulder) [reply varats/varats/experiments/vara/blame_report_experiment.py:124]: Ok, I'll keep the branch and just close the PR

**Comment by vulder** [comment]
Keep the changes alive but remove the PR

**Pull request closed without merging**